src/assets/javascript/application.js

```diff
@@ -1,84 +1,76 @@
+            n < t.length;
+            n++
+          )
+            "text" == t[n].type && (t[n].value = "");
         }
       }.bind(this)
     );
   });
 }
-
-var onIntNumberSelected = function (intPhoneNumberCheckbox, phoneNumberInput) {
-  if (intPhoneNumberCheckbox.checked) {
-    phoneNumberInput.value = "";
-    phoneNumberInput.disabled = true;
-    phoneNumberInput.classList.add("govuk-input--disabled");
-  } else {
-    phoneNumberInput.disabled = false;
-    phoneNumberInput.classList.remove("govuk-input--disabled");
-  }
+var onIntNumberSelected = function (e, t) {
+  e.checked
+    ? ((t.value = ""),
+      (t.disabled = !0),
+      t.classList.add("govuk-input--disabled"))
+    : ((t.disabled = !1), t.classList.remove("govuk-input--disabled"));
 };
-
 function initEnterPhoneNumber() {
-  var phoneNumberInput = document.querySelector("#phoneNumber");
-  var intPhoneNumberCheckbox = document.querySelector(
-    "#hasInternationalPhoneNumber"
-  );
-  if (phoneNumberInput && intPhoneNumberCheckbox) {
-    intPhoneNumberCheckbox.addEventListener("click", function (event) {
-      onIntNumberSelected(event.currentTarget, phoneNumberInput);
-    });
-    window.addEventListener(
-      "load",
-      onIntNumberSelected(intPhoneNumberCheckbox, phoneNumberInput)
-    );
-  }
+  var t = document.querySelector("#phoneNumber"),
+    e = document.querySelector("#hasInternationalPhoneNumber");
+  t &&
+    e &&
+    (e.addEventListener("click", function (e) {
+      onIntNumberSelected(e.currentTarget, t);
+    }),
+    window.addEventListener("load", onIntNumberSelected(e, t)));
 }
 
-(function (w) {
-  "use strict";
-  function appInit(trackingId, analyticsCookieDomain) {
-    window.GOVUKFrontend.initAll();
-    var cookies = window.GOVSignIn.Cookies(trackingId, analyticsCookieDomain);
+window.DI = window.DI || {};
+window.DI.analyticsUa = window.DI.analyticsUa || {};
 
-    if (cookies.hasConsentForAnalytics()) {
-      cookies.initAnalytics();
-    }
+(function (e) {
+  "use strict";
 
-    if (cookies.isOnCookiesPage()) {
-      cookies.cookiesPageInit();
-    } else {
-      cookies.cookieBannerInit();
-    }
-  }
+  function appInit() {
 
-  initFeedbackRadioButtons();
-  initEnterPhoneNumber();
+    window.GOVUKFrontend.initAll();
 
-  if (w.GOVUK && w.GOVUK.Modules && w.GOVUK.Modules.ShowPassword) {
-    var modules = document.querySelectorAll('[data-module="show-password"]');
+    var cookies = window.GOVSignIn.Cookies();
 
-    for (var i = 0, l = modules.length; i < l; i++) {
-      if (GOVUK.Modules.ShowPassword.prototype.init) {
-        new GOVUK.Modules.ShowPassword(modules[i]).init();
-      }
+    if (window.DI.analyticsGa4.cookie.hasConsentForAnalytics()) {
+      cookies.initAnalytics();
     }
   }
 
-  w.GOVSignIn.appInit = appInit;
+  if (
+    (initFeedbackRadioButtons(),
+    initEnterPhoneNumber(),
+    e.GOVSignIn && e.GOVSignIn.Modules && e.GOVSignIn.Modules.ShowPassword)
+  )
+    for (
```

Comment:
why have you change this part of the file? Don't see any relation with Google Analytics or the cookie banner?